src/main/asciidoc/entity-callbacks.adoc

```diff
@@ -0,0 +1,163 @@
+[entity-callbacks]
+= Entity Callbacks
+
+The Spring Data infrastructure provides hooks for modifying an entity before and/or after certain methods are invoked.
+Those so called ``EntityCallback``s provide a convenient way to check and potentially modify an entity in a callback fashioned style. +
```

Comment:
Please don't pluralize code objects in this way. Rather. refer to `EntityCallback` instances or `EntityCallback` objects or some similar word. (In a nut shell, use the class or interface name as an adjective for a noun.)